flareon/src/auth.rs

```diff
@@ -393,9 +400,10 @@ impl Debug for PasswordHash {
     }
 }
 
+const MAX_PASSWORD_HASH_LENGTH: u32 = 128;
```

Comment:
In the future, this should be a variable. Both Argon2 and PBKDF2 support variable-length hashes.

Reply:
True. I've set this to 128 because this is already more than enough for a 512-bit hash (encoded as base64) + some salt + some metadata. With currently used Argon2id, the total length of hash is less than 100 bytes.

As far as I'm concerned, there's little/no penalty between using, let's say, VARCHAR(128) and VARCHAR(256) in the database engines, so perhaps we can slightly increase this value for any hash size increases in the future? What do you think?

Reply:
Actually, there seems to be some impact on memory usage/performance: https://dba.stackexchange.com/a/76470 . Generally, I would argue we should use the smallest number we can (and Django uses 128, for instance).

Reply:
I'm convinced, let's keep it as it is